dotnet/src/SemanticKernel.UnitTests/Planning/PlanTests.cs

```diff
@@ -487,10 +487,9 @@ public async Task CanExecutePlanWithOneStepAndStateAsync()
             .Returns(() => Task.FromResult(returnContext));
 
         var plan = new Plan(mockFunction.Object);
-        plan.State.Set("input", "Cleopatra");
 
         // Act
-        var result = await plan.InvokeAsync();
+        var result = await plan.InvokeAsync(input: "Cleopatra");
```

Comment:
Shouldn't we also use `Kernel.RunAsync(plan, "Cleopatra");` ?

Reply:
yup, can do.

Reply:
oh, this is a UT that doesn't maintain a kernel instance (it has only a mock). In this case, since we're testing one specific piece of code, it's ok to use the lower-level .InvokeAsync() call rather than going through the kernel. That ensures that we're only testing this "unit" and not any potential problems the kernel is introducing as a wrapper.

Reply:
reverted (reduced the scope of the PR)